core/src/HIP/Kokkos_HIP_Parallel_Range.hpp

```diff
@@ -174,11 +174,14 @@ class ParallelReduce<FunctorType, Kokkos::RangePolicy<Traits...>, ReducerType,
   size_type* m_scratch_space = nullptr;
   size_type* m_scratch_flags = nullptr;
 
-  // FIXME_HIP_PERFORMANCE Need a rule to choose when to use shared memory and
-  // when to use shuffle
+#if HIP_VERSION < 401
```

Comment:
Shouldn't it be `410` for ROCm 4.1?

Reply:
No the current code is correct, I've checked.